images/windowsserver2022/scripts/setup-2.ps1

```diff
@@ -0,0 +1,12 @@
+Write-Host "Removing the WinHTTP proxy" 
+netsh.exe winhttp reset proxy
+
+Write-Host "Setting the disk SAN policy to Onlineall:" 
+
+"SAN Policy=OnlineAll" | diskpart 
```

Comment:
These scripts have a lot to grok in them. Some of the commands explicitly set values that are already the OS default values. Should we keep them to follow the doc, or skip because they're effectively a no-op and have a more straightforward sample?